### DISCUSSION TITLE
Using decorator in Modelx

### DISCUSSION THREAD

**Discussion by bakerwy** [fumitoh/modelx#170] [Q&A]
Hi,

I would like to use decorator in the modelx function in the model for cells, 
but I don't know how to do that. 

I know decorators can be applied by modifying __init__.py for each space manually, 
but I wonder if I can use it when I build model interactively in the terminal.

Thanks in advance.

**Comment by fumitoh** [comment]
Decorators cannot be used to define cells formulas. What’s the use case of a decorator?

**Reply by bakerwy** [reply]
I tried to use Numba for a special occasion.
But I resolve it by adding external function using Numba and import it in the model
